### ISSUE TITLE
Flickering black rectangles when useCanvas = false

### ISSUE THREAD

**Issue by avandecreme** [openseadragon/openseadragon#661]
Here is the result of using useCanvas=false on master:

https://docs.google.com/file/d/0BzAcRIdakTF0a2pJbkE4MTVGYzA/edit?usp=drivesdk

**Comment by iangilman** [comment]
Yow! Icky. Thanks for catching this.

**Comment by iangilman** [comment]
Hmm...I can't repro on my machine. What browser and OS are you using?

**Comment by avandecreme** [comment]
I reproduce on both Firefox and Chrome on Linux.

For some reason, I don't reproduce it using `grunt connect watch` but I do using either netbeans' internal web server or npm's `http-server`.
This is probably due to some cache policy but I don't see any header related to that. All I am seeing is that with grunt, I get 304 (Not Modified) responses from the server when a tile is requested twice but not with the other servers.

Anyway, I found a potential fix:
Replace https://github.com/openseadragon/openseadragon/blob/master/src/tile.js#L206 by:
```javascript
this.imgElement = this.cacheImageRecord.getImage().cloneNode();
```
I think the cloneNode is necessary in case the image is reused somewhere else but I did not check.
